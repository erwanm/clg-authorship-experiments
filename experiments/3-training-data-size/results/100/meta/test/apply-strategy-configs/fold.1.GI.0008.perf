0.461750	0.743200	0.621300
