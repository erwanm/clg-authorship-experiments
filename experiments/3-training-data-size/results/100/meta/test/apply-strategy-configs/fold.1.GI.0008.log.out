apply-config.sh: applying config '/tmp/expe3/100/process/selected-meta-configs/fold.1.0098.model/strategy-configs/fold.1.GI.0008' to cases '/home/moreaue/dump/expe3//results/100/meta/test/apply-strategy-configs/only-cases.tmp.L1MIpVmN'; results written to '/home/moreaue/dump/expe3//results/100/meta/test/apply-strategy-configs/fold.1.GI.0008'
apply-config.sh: done.
